apps/api/src/server/app/case-officer/case-officer.validators.js

```diff
@@ -1,4 +1,28 @@
+import { body } from 'express-validator';
+import { composeMiddleware } from '@pins/express';
+import { validationErrorHandler } from '../middleware/error-handler.js';
 import { difference } from 'lodash-es';
+import { validateAppealStatus } from '../middleware/validate-appeal-status.js';
+import { appealStates } from '../state-machine/transition-state.js';
+
+export const validateAppealBelongsToCaseOfficer = validateAppealStatus([
+	appealStates.received_lpa_questionnaire,
+	appealStates.incomplete_lpa_questionnaire
+]);
+
+export const validateAppealHasIncompleteQuestionnaire = validateAppealStatus([
+	appealStates.incomplete_lpa_questionnaire
+]);
```

Comment:
I feel like creating these separate validators is a little overkill for what it is

Reply:
The first one I think makes sense as otherwise the same appealStates are being added to each route so it's not really DRY... the second one is a bit meh as that is only used once, but I wanted to stick to a pattern...